### PR TITLE
Update whats_new.rst

### PR DESCRIPTION
**Pull request by jospoortvliet**
create a whats new for users.

images coming separately.

### CODE REVIEW

**Comment by nickvergessen** [comment]
Requires https://github.com/nextcloud/documentation/pull/480

File: user_manual/whats_new.rst [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+
+   .. figure:: images/contacts_popup.png
+      :alt: screenshot of popup over avatar
+
+4. Ability to sent multiple unique sharing links each with their own settings, by entering email addresses (the recipient will receive an email)
+
+   .. figure:: images/multi_sharing.png
+      :alt: screenshot of multiple sharing links
+
+5. Many other improvements and new apps, like screensharing in Video calls, new Circles app for user group handling, push notifications, notifications of file changes even when shared to another server, you can undo removal of files from a folder you shared even if the removal was done by a recipient, directly sharing to social media and much more.
```
Comment (MariusBluem) [inline user_manual/whats_new.rst]: circles is misunderstanding - "user-defined groups" sounds better IMHO

**Comment by jospoortvliet** [comment]
Anything else? If not, anyone - pls merge ;-)

**Comment by MorrisJobke** [comment]
backported to stable12

**Pull request merged**